en/docs/setup/migrating-what-has-changed.md

```diff
@@ -3,7 +3,7 @@
 WSO2 Identity Server 5.10.0 brings a range of new features and major improvements. In IS 5.10.0 following aspects has 
 been changed compared to the previous IS versions.
 
-## Handling Last login time as an identity claim
+## Handling Last login Time as an Identity Claim
```

Comment:
subheads should be in Sentence case and not Title Case. Therefore this heading should be **Handling 'last login time' as an identity claim**